src/main/java/io/pravega/connectors/flink/sink/PravegaSinkBuilder.java

```diff
@@ -47,6 +46,10 @@
     PravegaSinkBuilder() {
     }
 
+    static <T> PravegaSinkBuilder<T> builder() {
```

Comment:
This builder function should be in the `PravegaSink`, so that we can keep the API compatibility.

Reply:
Follow the builder pattern in source.